components/ListView/ListItemImages.js

```diff
@@ -21,30 +21,34 @@ class ListItemImages extends React.Component {
 
   // maps to Remove button for FAILED
   handleDelete() {
-    this.props.deletingCompose(this.props.listItem.id);
+    const { deletingCompose, listItem } = this.props;
+    deletingCompose(listItem.id);
```

Comment:
This is another such case. The extra aliasing adds no benefit at all here.

Reply:
Right, but I can't say that it really makes sense here. The [intention](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md) is for render methods that directly map property names to the rendered results, not for these helper methods. If it forces us to add such irrelevant and even dangerous noise, then I'd rather vote for disabling that eslint error.

Reply:
Agree with you. I'll disable this eslint rule and keep most of props at destructuring-assignment and make some of props back to `this.props` style.